ckanext/harvest/harvesters/base.py

```diff
@@ -21,16 +21,29 @@
 from ckan.plugins.core import SingletonPlugin, implements
 from ckanext.harvest.interfaces import IHarvester
 
+if p.toolkit.check_ckan_version(min_version='2.3'):
+    from ckan.lib.munge import munge_tag
```

Comment:
I did do that initially, but the problem turned out to be that although ckan 2.2 does have this function, it is a rubbish version and fails a couple of the tests e.g. turning 'river quality' into 'riverquality'.

If we ever were to take the function out of ckan then we can add a max_version here without too much trouble.